lib/app.js

```diff
@@ -41,11 +45,17 @@ module.exports = {
 
     if (args._ && args._.length > 0) {
       var command = args._[0];
-      // XXX: This is tricky. Because minimist treat first params after
-      // -r as value of r. But it is actually sourceDir to us.
-      var sourceDir = args.r ? args.r : args._[1];
-      var destDir = args.r ? args._[1] : args._[2];
-      var parseOption;
+      var sourceDir = null, destDir = null, parseOption;
+
+      console.log(args._.length);
+      console.log(args._);
```

Comment:
I think these two lines are for debugging purpose, right? If so, they should be removed.